syntaxes/ralph.tmLanguage.json

```diff
@@ -156,6 +172,16 @@
           "name": "support.function.builtIn.ralph"
         }
       ]
+    },
+    "global-types": {
+      "patterns": [
+        {
+          "comment": "types",
+          "name": "entity.name.type.rust",
```

Comment:
Should this be `entity.name.type.ralph`?

Reply:
indeed